.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.js filter=lfs diff=lfs merge=lfs -text
```

Comment:
Our own JavaScript and CSS should have normal version control.

Reply:
updated.. make sense, for now i removed css as we only have 2 css for travel and college pages. Rest other pages are using cdn version of materialize css.